### ISSUE TITLE
path error with basic example

### ISSUE THREAD

**Issue by blowsie** [typicode/json-server#420]
Hi, I tried to use the tool today and upon my first `POST` request the app crashes, every time.
I used the basic example, maybe there is a bug in the tool? 

Windows 10

```

POST /posts 201 38.992 ms - 65
path.js:9
    throw new TypeError('Path must be a string. Received ' +
    ^

TypeError: Path must be a string. Received null
```

![pathbug](https://cloud.githubusercontent.com/assets/308572/20604052/5acbc7fc-b25c-11e6-87e4-bf36dd6f4757.gif)

**Comment by blowsie** [comment]
I think the issue here was webstorm also running watches on the file

**Comment by typicode** [comment]
Hi @blowsie,

On OS X it works so it may also be related to the OS.
Anyway I think watch events could be better handled in the code, especially if there's some error.

Thanks for the report :)

**Comment by blowsie** [comment]
Doing a simple check if the file in the argument is null should prrvent this issue

**Comment by typicode** [comment]
Yes, `if (filename) { ...` do you want to make a PR?

**Comment by blowsie** [comment]
Can do, when i get time.

I stumbled across this today which would explain in a bit more detail why this happens.
https://webpack.github.io/docs/webpack-dev-server.html#working-with-editors-ides-supporting-safe-write

**Comment by typicode** [comment]
Got some time, so I've added a check.

**Comment by typicode** [comment]
I've published `v0.9.2` with the fix.
Let me know if it doesn't work.
